lib/mailer.js

```diff
@@ -103,6 +103,23 @@ module.exports = function (config, log) {
           }
         ))
       }
+      mailer.sendUnblockCode = function (account, unblockCode, opts) {
```

Comment:
Change the first parameter from `email` => `account` so that both the account email and uid can be passed to the mailer.